index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">    
+    <link rel="stylesheet" href="style.css">  
+     
+    <title>My Portfolio</title>
+</head>
+<body>
+    <div class="maincontainer" >       
+
+        <div class="mainmenu"> 
+
+            <div class="LogoPlaceholder"> <img src="img/Mylogo.png" alt="my logo">   </div>
+
+            <div class="itemnavigation">
+                <div class="navitems "><a href="#portfolio" > Portfolio</a> <br> </div> 
+                
+                <div class="navitems"><a href="#portfolio" > About</a> </div> 
+
+                <div class="navitems" ><a href="#portfolio" > contact</a> </div> 
+               
+            </div> 
+         </div>
+
+
+       
+       <div class="healdline">       
+        <img src="img/Headerbg.png" alt="my logo"> 
+            <br>
+            <br>
+                <div class="headlinebody">
+                    <br>
+                      <br> 
+                      <br>
+                      <br>                                   
+                  
+                    <div class="introductionclass">
+                      <p>I’m Samuel. <br>
+                        Glad to see you!</p>
+                    </div>
+                    <div class="second_primary_text">
+                        <p> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+
+                    </div>
+                    
+                        <div calss="letconnect">
+                            <p class="letconnect"> LET’S CONNECT</p>
+                        </div>
```

Comment:
- [ ] Kindly, try to reduce the use of `div` tags mostly above a single tag, please try to simplify your codes 👍 

example:
```
<p class="letconnect"> LET’S CONNECT</p>

```